### ISSUE TITLE
Some thoughts on problem 1

### ISSUE THREAD

**Issue by dylan1951** [rmsams/stablecoins#3]
I've been pondering the subjects in this paper for a while and I just fortunately stumbled upon it while exploring ideas with ChatGPT. It was a very interesting read and solidified and labeled some concepts in mind.

I wonder if there could exist a truely endogenous stabilisation model that is not tied to any external parameter like computation cost.

Something that comes to mind is bonds. In a currency without loans, savings account or interest rates and no risk of default, what determines the price of a bond? The only factor is **expected inflation**. If people expect more inflation to occur the nominal bond price will fall and vice versa.

In other words we can derive the expected changes in the real value of the currency from the nominal bond prices.

Could a cryptocurrency with some bond market use this data to decide the needed adjustments in supply to provide stability?

**Comment by dylan1951** [comment]
Maybe this could work if the supply had momentum/velocity. Then changes in bond prices (changes in expected inflation) could be used to adjust the rate/velocity of the supply.

If bond prices start to fall it signals that the rate of inflation is increasing, therefore the rate of supply should be lowered. If you continually make adjustments to the rate of supply, eventually the bond prices should follow a stable curve.

At this point you have achieved a constant rate of inflation.

Maybe to achieve 0% inflation you would set the coupon rate equal to the opportunity cost?

**Comment by dylan1951** [comment]
Actually the bond prices probably wouldn't be able to account for short term fluctuations very well.

Maybe the solution is a hybrid approach of using computational power to account for short time price fluctuations, combined with the aforementioned bond rate method to account for increasing hardware efficiency, instead of a Moore’s Law assumption.

Using this method the coins purchasing power could be totally stable even very far into the future where hardware efficiencies are unknown with quantum computing etc.

**Comment by sambacha** [comment]
why should society (re: the respective community) guarantee your purchasing power?

**Comment by dylan1951** [comment]
> why should society (re: the respective community) guarantee your purchasing power?

Not sure I follow exactly, but if a decentralised currency protocol can provide stable purchasing power then that is very useful is it not? If it's possible it will happen regardless of what society ought to do.
